index.d.ts

```diff
@@ -198,6 +202,22 @@ export interface TopicMetadata {
  */
 export type MessageRouter = (message: Message, topicMetadata: TopicMetadata) => number;
 
+export interface EncryptionKey {
+  key: string;
+  value: string;
```

Comment:
The value should be a byte array. In C++, `std::string` could also used as a byte array, but in Node.js, `Buffer` should be used as a byte array.

Sorry for not documenting which fields should represent byte array in `EncryptionContext`, but I just checked the implementation and confirmed the following fields should be bytes rather than strings:
- `EncryptionKey::value`
- `param`